logviewer/src/main/java/org/togetherjava/logwatcher/entities/InstantWrapper.java

```diff
@@ -0,0 +1,68 @@
+package org.togetherjava.logwatcher.entities;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.text.SimpleDateFormat;
+import java.time.Instant;
+import java.util.Date;
+
+public class InstantWrapper {
```

Comment:
What is this for?

Reply:
When an Event comes from the Bot, I can't directly parse an Instant from the JSON. 
So I added an little Record Class that can hold the values for the Instant and convert it after it arrived

Reply:
I see. If you feel like it you can add a small javadoc comment but it's not really important.